velox/functions/prestosql/tests/SequenceTest.cpp

```diff
@@ -207,16 +207,41 @@ TEST_F(SequenceTest, dateInvalidIntervalDayStep) {
       expected);
 }
 
+TEST_F(SequenceTest, invalidDate) {
```

Comment:
Why do you say "invalid"?

Reply:
This test case block should've gone away. Sorry, I used it locally and forgot to remove.